### DISCUSSION TITLE
Whether already exist tool change mitsuba3 xml to mitsuba0.5 xml in mitsuba-blender？

### DISCUSSION THREAD

**Discussion by xdobetter** [mitsuba-renderer/mitsuba-blender#60] [Q&A] [answered]
Thanks great work.I want change format mitsuba3 xml to mitsuba0.5 xml.There whether exist already the method,if not,I maybe try change format according mitsuba0.6 xml document.

**Comment by merlinND** [comment, marked as answer]
Hello @xdobetter,

I don't think that such a tool exists, and we haven't really had a need for it because the formats for 0.5, 0.6 and 3.0 are very similar, so fixes can typically be applied by hand.
Moreover, most incompatibilities will be due to plugins that haven't been implemented in Mitsuba 3, so an automatic conversion wouldn't be able to fix that.

I think the easiest way is to try loading the scene in Mitsuba 3, checking the error message, applying the fixes, and repeating until the scene loads.
It's a bit of manual work, but in any case you would have to decide for  yourself what to replace the missing plugins with.

I hope this helps!

**Reply by xdobetter** [reply]
Thank your advice very much!
